pymatgen/core/composition.py

```diff
@@ -25,7 +25,10 @@
 if TYPE_CHECKING:
     from collections.abc import Generator, Iterator
 
-SpeciesLike = Union[str, Element, Species, DummySpecies]
```

Comment:
Make sure this `SpeciesLike` is the same as `from pymatgen.util.typing import SpeciesLike`

Reply:
Confirmed: https://github.com/materialsproject/pymatgen/blob/b1e50237cfd90a68440fb82806b58e59db2a4d5e/pymatgen/util/typing.py#L24